scripts/dnu.sh

```diff
@@ -8,5 +8,7 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
+# work around restore timeouts on Mono
+export MONO_THREADS_PER_CPU=50
```

Comment:
Also one more thing, we need to not override this if people have already set it.